xml/System.Web/SameSiteMode.xml

```diff
@@ -14,7 +14,20 @@
   </Base>
   <Docs>
     <summary>Specifies constants that indicate the value for the SameSite attribute of the cookie.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
```

Comment:
Reorder:
- The behavior of SameSiteMode.None was modified by patches described in these KBs...
- Before, https://tools.ietf.org/html/draft-west-first-party-cookies-07#section-4.1
- After, https://tools.ietf.org/html/draft-west-cookie-incrementalism-00